src/test/scala/mesosphere/marathon/upgrade/TaskStartActorTest.scala

```diff
@@ -92,7 +93,7 @@ class TaskStartActorTest
     when(f.launchQueue.get(app.id)).thenReturn(None)
     val task =
       MarathonTestHelper.startingTaskForApp(app.id, appVersion = Timestamp(1024))
-    f.taskCreationHandler.created(TaskStateOp.LaunchEphemeral(task)).futureValue
+    f.taskCreationHandler.created(TaskStateOp.LaunchEphemeral(task)).isReadyWithin(Span(1, Second))
```

Comment:
`futureValue` was confusing me when I read it. I guess it depends if this is more part of the setup as in `given f.taskCreationHandler.created(TaskStateOp.LaunchEphemeral(task)) is ready` or if it is more of a assertion as in `should be ready within(5 seconds)`.

I'm fine with both :)